packages/babel-plugin-transform-wpcalypso-async/jest.config.js

```diff
@@ -2,4 +2,5 @@ module.exports = {
 	preset: '@automattic/calypso-build',
 	rootDir: __dirname,
 	cacheDirectory: '<rootDir>/../../.cache/jest',
+	resolver: '<rootDir>/../../test/module-resolver.js',
```

Comment:
This would be nice to move to some shared preset. The packages' Jest configs are getting repetitive: the `cacheDirectory` is always the same, and now the `resolver` field is, too.

By the way, even the `rootDir` field could be removed one day, when [this Jest bug](https://github.com/facebook/jest/issues/8293) gets fixed.

Reply:
I couldn't figure out how to do it. When I added the `cacheDirectory` to the base jest config I got some problems relative to how jest resolves paths... I don't remember the details, I'll try again.

Reply:
Created a separate PR for it: #46953 46953